mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -211,9 +211,10 @@ def Rock_GeneralGemmParamsAttr : Rock_Attr<"GeneralGemmParams", [RockTuningParam
 
   let extraClassDeclaration = [{
     std::string getPerfConfigStr(){
-      const Twine paramsConcat = Twine(getMPerBlock()) + ","
-      + Twine(getNPerBlock()) + ","
+      const Twine paramsConcat = Twine(64) + "," /* block size */
```

Comment:
The perf config has a block size? Is that part of the tuning space?

Reply:
Actually I was going to ask this but just forgot. Let me check with the others.